recipe/meta.yaml

```diff
@@ -1,38 +1,52 @@
-{% set name = "perl-sys-cpu" %}
 {% set version = "0.61" %}
-{% set hash = "250a86b79c231001c4ae71d2f66428092a4fbb2070971acafd471aa49739c9e4" %}
+{% set sha256 = "250a86b79c231001c4ae71d2f66428092a4fbb2070971acafd471aa49739c9e4" %}
  
 package:
-    name: {{ name }}
-    version: {{ version }}
-    
+  name: perl-sys-cpu
+  version: {{ version }}
+
 source:
-    url: https://cpan.metacpan.org/authors/id/M/MZ/MZSANFORD/Sys-CPU-{{ version }}.tar.gz
-    sha256: {{ hash }}
-    
+  - url: https://cpan.metacpan.org/authors/id/M/MZ/MZSANFORD/Sys-CPU-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+  - url: https://raw.githubusercontent.com/Perl/perl5/v5.32.1/Copying
+    sha256: d77d235e41d54594865151f4751e835c5a82322b0e87ace266567c3391a4b912
+  - url: https://raw.githubusercontent.com/Perl/perl5/v5.32.1/Artistic
+    sha256: dd90d4f42e4dcadf5a7c09eea0189d93c7b37ae560c91f0f6d5233ed3b9292a2
+
 build:
-    number: '2'
-    skip: true  # [win]  
-    skip: true  # [osx]
-    
+  number: 2
+  script:
+    - perl Makefile.PL INSTALLDIRS=vendor NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
```

Comment:
`conda-build` does not check each individual build/script command for failure on Windows.
=> `make test` actually fails even though the build passes.
I'll try nmake next